Controllers/UsersController.cs

```diff
@@ -21,7 +22,7 @@ public class UsersController : ControllerBase
 
         private IUserService _userService;
         JWTHelper jwtHelper = new JWTHelper();
-        //RefreshTokenHelper refToken = new RefreshTokenHelper();
+        RefreshTokenHelper refToken = new RefreshTokenHelper();
```

Comment:
Move this line inside the "if condition" on line 40

Reply:
done.